website/src/routes/api/(schemas)/string/index.mdx

```diff
@@ -2,6 +2,116 @@
 title: string
 ---
 
+import { Property } from '~/components';
+import { Link } from '@builder.io/qwik-city';
+
 # string
 
-> The content of this page is not yet ready. Until then just use the [source code](https://github.com/fabian-hiller/valibot/blob/main/library/src/schemas/string/string.ts).
+Schema that parses the input to string.
+
+```ts
+const schema = string(errorMessage, pipe);
+```
+
+## Parameter
+
+- `errorMessage` <Property type='string'/> = 'Invalid type'
+- `pipe` <Property type={{ type: 'custom', name: 'Pipe<string>' }}/> = []
+
+## Return
+
+- `schema` <Property type={{ type: 'custom', name: 'StringSchema' }}/>
+
+## Examples
+
+```ts
+const schema = string();
+parse(schema, 'string'); // returns 'string'
+parse(schema, 123); // throw ValiError 'Invalid Type'
+
+const schemaWithCustomErrMsg = string('custom error message');
+parse(schemaWithCustomErrMsg, 123); // throw ValiError 'custom error message'
+
+string([maxLength(5)]);
+string([minLength(5)]);
+string([length(5)]);
+string([email()]);
+string([url()]);
+string([emoji()]);
+string([uuid()]);
+string([cuid2()]);
+string([ulid()]);
+string([regex(regex)]);
+string([includes(string)]);
+string([startsWith(string)]);
+string([endsWith(string)]);
+string([datetime()]);
+string([ip()]);
+```
+
+## Methods
+
+- <Link href="/api/brand">`brand`</Link>
+- <Link href="/api/coerce">`coerce`</Link>
+- <Link href="/api/coerceAsync">`coerceAsync`</Link>
+- <Link href="/api/fallback">`fallback`</Link>
+- <Link href="/api/fallbackAsync">`fallbackAsync`</Link>
+- <Link href="/api/getDefault">`getDefault`</Link>
+- <Link href="/api/is">`is`</Link>
+- <Link href="/api/parse">`parse`</Link>
+- <Link href="/api/parseAsync">`parseAsync`</Link>
+- <Link href="/api/safeParse">`safeParse`</Link>
+- <Link href="/api/safeParseAsync">`safeParseAsync`</Link>
+- <Link href="/api/transform">`transform`</Link>
+- <Link href="/api/transformAsync">`transformAsync`</Link>
+- <Link href="/api/unwrap">`unwrap`</Link>
+
+## Transformations
+
+- <Link href="/api/toCustom">`toCustom`</Link>
+- <Link href="/api/toCustomAsync">`toCustomAsync`</Link>
+- <Link href="/api/toLowerCase">`toLowerCase`</Link>
+- <Link href="/api/toMaxValue">`toMaxValue`</Link>
+- <Link href="/api/toMinValue">`toMinValue`</Link>
+- <Link href="/api/toTrimmed">`toTrimmed`</Link>
+- <Link href="/api/toTrimmedEnd">`toTrimmedEnd`</Link>
+- <Link href="/api/toTrimmedStart">`toTrimmedStart`</Link>
+- <Link href="/api/toUpperCase">`toUpperCase`</Link>
+
+## Validations
```

Comment:
![badge](https://img.shields.io/badge/review-memo-blue.svg)
Validations which can be used with string()

Reply:
Yes, thats great! Good idea!